websites/M/Maki.gg/presence.ts

```diff
@@ -136,5 +150,6 @@ presence.on("UpdateData", async () => {
   if (!presenceData.details) {
     presence.setTrayTitle();
     presence.setActivity();
-  } else presence.setActivity(presenceData);
+  } else
+    presence.setActivity(presenceData);
```

Comment:
You are not required to have this check, alternatively, you can just do this: 
```suggestion
presence.setActivity(presenceData)
````

Reply:
Ah I see. Thank you